homework-submission/todo_react/src/TodoLists.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "./TodoLists.css";
+
+
+
+export default class TodoLists extends React.Component {
+
+  render() {
+
+    let { description, deadline, done, id, handleClick, removeItem } = this.props;
+
+    // console.log(data[2].deadline)
+
+    return (
+      <div className="list">
+
+        <ul>
+          <li>
```

Comment:
This is a `ul` element iterated by the `.map` method, which means you are repeating this element for multiple times giving you a multiple `ul`s with one child `li` in each.